tasks/build.yml

```
@@ -0,0 +1,38 @@
+---
+# This is included in deploy.yml only if {{ librespot_binary_url }} is not defined
+
+- name: Download librespot archive
+  get_url:
+    url: "{{ librespot_archive_url }}"
+    dest: "{{ global_cache_dir }}/librespot-{{ librespot_version }}.tar.gz"
+  become: false
+  delegate_to: localhost
```

Please add `run_once: true` so that the file will only be downloaded once, even when deploying multiple hosts.

```suggestion
  delegate_to: localhost
  run_once: true
```

---

`get_url` has `force: false` by default, so the file will only be downloaded once. (That holds if `dest` is a file, which is the case here.)